README.md

```diff
@@ -257,6 +257,29 @@ If a job script for your machine is not available, try modifying the default
 job script in `configs/job_script.default.bash` or one of the job scripts for
 another machine to fit your needs.
 
+## Customizing plots or creating new ones
+
+There are three main ways to either customize the plots that MPAS-Analysis
+already makes or creating new ones:
+
+1. customize the config file. Some features, such as colormaps and colorbar
+   limits for color shaded plot or depth ranges for ocean region time series,
+   can be customized: look at `mpas_analysis/config_default` for available
+   customization for each analysis task.
+2. read in the metadata computed by MPAS-Analysis into custom scripts. When
```

Comment:
```suggestion
2. read in the analysis data computed by MPAS-Analysis into custom scripts. When
```
metadata is the data withing a file that describes the data (so the attributes and such).